modules/local/fusioninspector/main.nf

```diff
@@ -41,6 +41,9 @@ process FUSIONINSPECTOR {
         --out_prefix $prefix \\
         --vis $args $args2
 
+    # Create the TSV file to avoid output file missing error when the TSV file hasn't been created (when no fusions have been found)
```

Comment:
I think you'll have to do the same for the other outputs, as it now fails with:
`Caused by:
  Missing output file(s) `*html` expected by process `RNAFUSION:FUSIONINSPECTOR_WORKFLOW:FUSIONINSPECTOR (test)``

Reply:
Okay thanks for testing!

Reply:
It should be okay now!